src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonListener.cs

```diff
@@ -33,6 +36,7 @@ public SingletonListener(MethodInfo method, SingletonAttribute attribute, Single
             _lockId += ".Listener";
 
             _trace = trace;
+            _logger = loggerFactory?.CreateLogger(LoggingCategories.Singleton);
```

Comment:
It didn't really fit in the other buckets. We may need a "Host.General" or something for messages that the host writes during normal execution like this. But since this was the only thing that fit that category, I decided to give it its own category.

These categories are one thing that we need to iterate on -- they don't have to be perfect right now.